validator/sawtooth_validator/journal/genesis.py

```diff
@@ -135,8 +135,7 @@ def start(self, on_done):
             for batch in genesis_data.batches:
                 scheduler.add_batch(batch)
 
-            self._transaction_executor.execute(scheduler,
-                                               require_txn_processors=True)
+            self._transaction_executor.execute(scheduler)
```

Comment:
This change should be part of the previous commit where this signature changed.  Separately, the previous commit is broken.